client/lib/analytics/statsd-utils.ts

```diff
@@ -1,7 +1,7 @@
 import config from '@automattic/calypso-config';
 
 function generateUrl( featureSlug: string, eventType: string, eventValue: string ) {
-	const slug = featureSlug.replace( /[.:-]/g, '_' );
+	const slug = featureSlug.replace( /[:-]/g, '_' );
```

Comment:
This changes so that we can include periods in the "slug", which lets us filter things in grafana.

Reply:
The `u` parameter has been for the page URL, though some where along the way Calypso started using it for different things.  Either way, it already has periods in it and should be URL encoded.

Reply:
Updated the PR to canonicalize the section name as the `u` parameter. (It's now explicitly called `calypsoSection` rather than featureSlug`) As a result, we can record multiple beacons in the same section simultaneously